sdk/communication/azure-communication-identity/src/test/java/com/azure/communication/identity/CommunicationIdentityAsyncTests.java

```diff
@@ -261,15 +311,15 @@ public void issueToken(HttpClient httpClient) {
         StepVerifier.create(
             asyncClient.createUser()
                 .flatMap(communicationUser -> {
-                    List<String> scopes = new ArrayList<>(Arrays.asList("chat"));
+                    List<CommunicationTokenScope> scopes = 
+                        new ArrayList<>(Arrays.asList(CommunicationTokenScope.CHAT));
                     return asyncClient.issueToken(communicationUser, scopes);
                 }))
             .assertNext(issuedToken -> {
                 assertNotNull(issuedToken.getToken());
                 assertFalse(issuedToken.getToken().isEmpty());
-                assertNotNull(issuedToken.getExpiresOn());
-                assertFalse(issuedToken.getExpiresOn().toString().isEmpty());
-                assertNotNull(issuedToken.getUser());
```

Comment:
keep this line?

Reply:
the issuedToken is updated to type `AccessToken` and no longer contains the user.